File: array/1672.richest_customer_wealth.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        mx = 0
        for ith in accounts:
            i_mx = 0
            for jth in ith:
                i_mx += jth
            mx = max(i_mx, mx)
        return mx

File: array/1672_richest_customer_wealth.cpp
#include <bits/stdc++.h>
using namespace std;

class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int mx = 0;
        for (auto ith: accounts) {
            int i_mx = 0;
            for (auto jth: ith) {
                i_mx += jth;
            }
            mx = max(i_mx, mx);
        }
        return mx;
    }
};

int main() {
    
}
// Thursday, December 23, 2021 | 07:56:57 PM (+06)
